pkg/url/url.go

```diff
@@ -92,6 +93,66 @@ func List(client *occlient.Client, componentName string, applicationName string)
 	return urlList, nil
 }
 
+// List returns all URLs for given component. The results can further be narrowed
+// down if a component name is provided, which will only list URLs for the
+// given component
```

Comment:
```go
// List returns all URLs for given component.
``` 
and 
```go
// The results can further be narrowed
// down if a component name is provided, which will only list URLs for the
// given component
```
sound like the same thing to me. Am I missing something here?

Reply:
You are right, the comment is confusing. I've updated it.